examples/microprofile/oci-tls-certificates/etc/unsupported-cert-tools/config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The caller should update these accordingly
+export COMPARTMENT_OCID=ocid1.tenancy.oc1..
```

Comment:
How about making command line arguments instead of sourcing?

Reply:
Filed https://github.com/helidon-io/helidon/issues/7507 as a follow-up enhancement for this and the next comment.